lib/extractSignatureHeaderAsDict.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+function extractSignatureHeaderAsDict ({ headers: { signature = null } }) {
```

Comment:
I think we probably want to return / throw an error here if signature doesn't exist.

Reply:
Ed mentioned that there may be cases that we want this plugin configurable to respond with `404` in all error cases as a security-by-obscurity measure. I figured on this "lighter" implementation until I had a better sense of how that would work.
I'll add error handling here.

Reply:
I think we can throw here. We can catch the error in the plugin where we invoke these methods and handle the error in a configurable way there.